imu_sensor_notes.md

```diff
@@ -94,13 +94,13 @@ The `cal_acc_horizontal_offset` is the offest (difference) that the Joy-Con/Pro-
 
 The `cal_acc_horizontal_offset` is always the same. Advise [here](spi_flash_dump_notes.md#6-axis-and-stick-device-parameters) for the values each model (JC Left, JC Right, Pro).
```

Comment:
`spi_flash_dump_notes.md#6-axis-and-stick-device-parameters`
to
`spi_flash_notes#6-axis-and-stick-device-parameters`